core/controllers/base.py

```diff
@@ -195,6 +195,7 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
         self.is_super_admin = (
             current_user_services.is_current_user_super_admin())
 
+        self.values['additional_angular_modules'] = []
```

Comment:
Can we do a similar thing for `iframed`?

Reply:
Yes! PTAL below